File: hello-world-scripts/hello.java
// Hello World in Java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class hello {
    public static void main(String[] args) {
        System.out.println("Hello, World!");

        // With additional formatting
        System.out.println("========================================");
        System.out.println("  Java Hello World");
        System.out.println("========================================");

        LocalDateTime now = LocalDateTime.now();
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
        System.out.println("Executed at: " + now.format(formatter));
    }
}

File: hello-world-scripts/hello.c
#include <stdio.h>
#include <time.h>

// Hello World in C
int main() {
    printf("Hello, World!\n");

    // With additional formatting
    printf("========================================\n");
    printf("  C Language Hello World\n");
    printf("========================================\n");

    time_t now = time(NULL);
    printf("Executed at: %s", ctime(&now));

    return 0;
}
